### DISCUSSION TITLE
Nightly build

### DISCUSSION THREAD

**Discussion by kostmick** [Open-Shell/Open-Shell-Menu#1100] [General]
How to download nightly build if constantly drops out https://ci.appveyor.com/project/passionate-coder/open-shell-menu/branch/master/artifacts 
{"message":"Artifacts download limit (1024 MB/day) exceeded."}
Actions?

**Comment by hooddy** [comment]
try tomorrow)

**Comment by ge0rdi** [comment]
@kostmick 
Thank you for reporting this.
Apparently we will have to find better storage for nightly builds as AppVeyor seems to have quite tight limits.

**Reply by blackcrack** [reply]
maybe let automatic upload to a free ftp.. like Libreoffice or Distributions of linux 
per %versiondirectory% then be all fully store and no download limit..

**Reply by kostmick** [reply]
Pls

**Comment by Ibuprophen** [comment]


**Reply by blackcrack** [reply]
the original File is them of the original Repository
and this is at moment 4.4.173 !!
https://ci.appveyor.com/project/passionate-coder/open-shell-menu/build/artifacts
and not from an foreign Repository @Ibuprophen and also not from your Repository Ibu !!
best

(edit) your offer is may in best will, but a wrong help.. original way is original way.. 
i like you @Ibuprophen therefore, you want helping, i see that , but this is not the right way.. \*hug*

**Comment by ge0rdi** [comment]
> I had uploaded the v4.4.200 to my File Host Account for others to test to see if the installation error is resolved (hopefully).

Official versions are available via GitHub releases page (or AppVeyor in case of nightly builds).
We do not recommend to download/install from unofficial sources.

**Reply by blackcrack** [reply]
at AppVeyor :  it is maybe possible for upload these files automatical like
 Libreoffice,virtualbox, OwnCloud/Nextcloud or Distributions of linux per %versiondirectory% then be all fully store and no download limit..
and the community can roll on like a "wz" community with 0-day files ;)  
this gives the gimmick to have a larger group the possible for download nightly builds.
and you see it at moment, it need some possible, because.. 2-3 minutes behind 12:00pm is the download fast exhausted..

or it is maybe the possibility give to let directly the nightly builds in a folder here in github and let ever delete the older files if over limit ? this could support to have ever the last 2-3 versions for all downloadable unlimited .... 

best :)

**Reply by kostmick** [reply]
@blackcrack It's better to directly host nightly builds in a folder here on github

**Reply by blackcrack** [reply]
yes, was a suggestion.. if this works to set up a max size and remove the older files and let upload ever the fresh builded files
would be nice .. ...@kostmick, what is your HeadProfile, would be nice if you post with them.. a totally new and fresh Profile is less trustful..
best

**Reply by kostmick** [reply]
@blackcrack 
https://github.com/kostmick

**Comment by ge0rdi** [comment]
@kostmick 
Nightly builds are now on Github releases page:
https://github.com/Open-Shell/Open-Shell-Menu/releases

**Reply by kostmick** [reply]
@ge0rdi 
Thank you, for your assistance
